profiles/logging.py

```diff
@@ -5,6 +5,9 @@
 
 from pythonjsonlogger import jsonlogger
 
+sql_engine_logger = logging.getLogger('sqlalchemy.engine')
+app_logger = logging.getLogger(__name__)
```

Comment:
Unrelated: the global variables are not used so unclear whether they do need to be global, perhaps to be moved to another PR

Reply:
I have reverted this change.